views/js/pciCreator/ims/audioRecordingInteraction/imsPciCreator.js

```diff
@@ -87,7 +89,8 @@ define([
 
                 displayDownloadLink: false,
                 updateResponsePartially: true,
-                partialUpdateInterval: 1000
+                partialUpdateInterval: 1000,
+                sequential: false
```

Comment:
If I'm not mistaken, this is not needed, because at the moment `sequential` is class, not a `property`. So this will always be `false`. No harm in leaving it though.

[Or if we change `sequential` to `property`, that would suit this PCI well]